src/main/java/org/jabref/gui/entryeditor/citationrelationtab/CitationRelationsTab.java

```diff
@@ -196,79 +199,101 @@ private SplitPane getPaneAndStartSearch(BibEntry entry) {
+                        if (selected) {
+                            addToggle.setGraphic(IconTheme.JabRefIcons.ADD_FILLED.withColor(IconTheme.SELECTED_COLOR).getGraphicNode());
+                        } else {
+                            addToggle.setGraphic(IconTheme.JabRefIcons.ADD.getGraphicNode());
+                        }
+                    });
+                    addToggle.getStyleClass().add("addEntryButton");
+                    addToggle.selectedProperty().bindBidirectional(listView.getItemBooleanProperty(entry));
+                    vContainer.getChildren().add(addToggle);
+                }
+
+                if (entry.entry().getDOI().isPresent() || entry.entry().getField(StandardField.URL).isPresent()) {
+                    Button openWeb = IconTheme.JabRefIcons.OPEN_LINK.asButton();
+                    openWeb.setTooltip(new Tooltip(Localization.lang("Open URL or DOI")));
+                    openWeb.setOnMouseClicked(event -> {
+                        String url = entry.entry().getDOI().flatMap(DOI::getExternalURI).map(URI::toString)
                                               .or(() -> entry.entry().getField(StandardField.URL)).orElse("");
-                            if (StringUtil.isNullOrEmpty(url)) {
-                                return;
-                            }
-                            try {
-                                JabRefDesktop.openBrowser(url, preferencesService.getFilePreferences());
-                            } catch (IOException ex) {
-                                dialogService.notify(Localization.lang("Unable to open link."));
-                            }
-                        });
-                        vContainer.getChildren().addLast(openWeb);
-                    }
-
-                    hContainer.getChildren().addAll(entryNode, separator, vContainer);
-                    hContainer.getStyleClass().add("entry-container");
-
-                    return hContainer;
-                })
-                .withOnMouseClickedEvent((ee, event) -> {
-                    if (!ee.isLocal()) {
-                        listView.getCheckModel().toggleCheckState(ee);
-                    }
-                })
-                .withPseudoClass(entrySelected, listView::getItemBooleanProperty)
-                .install(listView);
-
-        listView.setSelectionModel(new NoSelectionModel<>());
-    }
+                        if (StringUtil.isNullOrEmpty(url)) {
+                            return;
+                        }
+                        try {
+                            JabRefDesktop.openBrowser(url, preferencesService.getFilePreferences());
+                        } catch (IOException ex) {
+                            dialogService.notify(Localization.lang("Unable to open link."));
+                        }
+                    });
+                    vContainer.getChildren().add(openWeb);
```

Comment:
Why was `addLast` changed to `openWeb`.

Maybe, you should provide screenshots before and after.